app/uk/gov/hmrc/incorporatedentityidentificationfrontend/models/PageConfig.scala

```diff
@@ -26,16 +26,15 @@ case class PageConfig(optServiceName: Option[String],
 
 object PageConfig {
 
-  def apply(optServiceName: Option[String],
-            optWelshServiceName: Option[String],
-            deskProServiceId: String,
+  def apply(deskProServiceId: String,
             signOutUrl: String,
-            accessibilityUrl: String
-           ) : PageConfig = {
+            accessibilityUrl: String,
+            labels: JourneyLabels
+           ): PageConfig = {
 
-    val labels: Option[JourneyLabels] = optWelshServiceName.map(JourneyLabels(_))
+    val optLabels = if (labels.nonEmpty) Some(labels) else None
 
-    new PageConfig(optServiceName, deskProServiceId, signOutUrl, accessibilityUrl, labels)
+    new PageConfig(labels.optEnglishServiceName, deskProServiceId, signOutUrl, accessibilityUrl, optLabels)
```

Comment:
As this is only used in the create config page could we have the optServiceName set as None, instead of labels.optEnglishServiceName as it's then being stored twice in each field. Not really a big issue though

Reply:
Yes, another good spot, that's unnecessary to pass it twice.